src/components/Profile/components/EmergencyInfoList/index.js

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+import './index.css';
+import ProfileInfoListItem from '../ProfileInfoListItem';
+import user from 'services/user';
+
+import { Typography, Grid, Card, CardHeader, CardContent, List } from '@material-ui/core';
+import { useEffect } from 'react';
```

Comment:
This can be merged onto line 1 with: `import { useState, useEffect } from 'react';`